pkg/util/proto/openapi.go

```diff
@@ -55,6 +55,7 @@ type SchemaVisitor interface {
 	VisitMap(*Map)
 	VisitPrimitive(*Primitive)
 	VisitKind(*Kind)
+	VisitArbitrary(*Arbitrary)
```

Comment:
Expanding this interface causes a ton of code to break in kubernetes/kubernetes...

https://github.com/kubernetes/kubernetes/pull/57059